src/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.HandleArray.cs

```diff
@@ -172,7 +175,8 @@ internal static void ApplyObjectToEnumerable(object value, JsonSerializerOptions
                 }
                 else
                 {
-                    ((IList)frame.JsonPropertyInfo.GetValueAsObject(frame.ReturnValue)).Add(value);
+                    IList list = (IList)frame.JsonPropertyInfo.GetValueAsObject(frame.ReturnValue);
```

Comment:
Is it fine to silently ignore the `value` if `list` is null?

Reply:
This was refactored to remove the ? check. Thanks